EIPS/eip-1822.md

```diff
@@ -74,12 +74,12 @@ The Proxy Contract proposed here should be deployed _as is_, and used as a drop-
 
 The proposed fallback function follows the common pattern seen in other Proxy Contract implementations such as [Zeppelin][1] or [Gnosis][2].
 
-However, rather than forcing use of a variable, the address of the Logic Contract is stored at the defined storage position `keccak256("PROXIABLE")`. This eliminates the possibility of collision between variables in the Proxy and Logic Contracts, thus providing "universal" compatibility with any Logic Contract.
+However, rather than forcing use of a variable, the address of the Logic Contract is stored at the defined storage position `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` which is obtained as `bytes32(uint256(keccak256('eip1967.proxy.implementation')) - 1)`. This reduces the possibility of collision between variables in the Proxy and Logic Contracts, thus providing "universal" compatibility with any Logic Contract. The use of this particular storage position is futher discussed on the page for the [EIP 1967](https://eips.ethereum.org/EIPS/eip-1967).
```

Comment:
```suggestion
However, rather than forcing use of a variable, the address of the Logic Contract is stored at the defined storage position `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` which is obtained as `bytes32(uint256(keccak256('eip1967.proxy.implementation')) - 1)`. This reduces the possibility of collision between variables in the Proxy and Logic Contracts, thus providing "universal" compatibility with any Logic Contract. The use of this particular storage position is further discussed on the page for the [EIP 1967](https://eips.ethereum.org/EIPS/eip-1967).
```